lib/ts/recipe/recipeRouter/index.tsx

```diff
@@ -24,7 +24,7 @@ export abstract class RecipeRouter {
         }
 
         const dynamicLoginMethods = Multitenancy.getInstanceOrThrow().getLoadedDynamicLoginMethods();
```

Comment:
We could check if we are using dynamic login methods before getting them. If we don't we could just use the old logic.